src/bioregistry/data/bioregistry.json

```diff
@@ -119164,6 +119164,7 @@
       "homepage": "http://www.wormbase.org",
       "miriam": "wb",
       "name": "Wormbase Gene ID",
+      "pattern": "^(CE[0-9]{5}|WB[A-Z][a-z]+\\d+)$",
```

Comment:
@muffymn this is not the right place to put this pattern since it is inside the `wikidata` block. It would have to be one level above this, directly under the `wormbase` entry.